presto-main/src/main/java/com/facebook/presto/testing/TestingConnectorSession.java

```diff
@@ -77,7 +77,7 @@ public String getQueryId()
     }
 
     @Override
-    public Identity getIdentity()
```

Comment:
I just wanted to keep the naming consistent with other method from ConnectorSession. There is `getProperty` method, but not `getConnectorProperty`. If you insist - i can change it. But i think it is kind of tautology, `ConnectorSession#getConnectorIdentity`.